.env

```diff
@@ -0,0 +1,3 @@
+MONGO_URL = 'mongodb+srv://noveltyUser:bwb6FhIxLTk7trUx@cluster0.oajux.mongodb.net/is-happy-thoughts?retryWrites=true&w=majority'
+
+RESET_DB = true
```

Comment:
You have forgot to list your .env to the gitnore file and are therefore sharing sensitive data with the world 🌎😬